src/CoaddPsf.cc

```diff
@@ -221,6 +221,30 @@ void addToImage(
 }
 
 
+afw::geom::Box2I CoaddPsf::doComputeBBox(
+    afw::geom::Point2D const & ccdXY,
+    afw::image::Color const & color
+) const {
+    afw::table::ExposureCatalog subcat = _catalog.subsetContaining(ccdXY, *_coaddWcs, true);
+    if (subcat.empty()) {
+        throw LSST_EXCEPT(
+            pex::exceptions::InvalidParameterError,
+            (boost::format("Cannot compute BBox at point %s; no input images at that point.")
+             % ccdXY).str());
+    }
+
+    afw::geom::Box2I overallBBox;
+    for (afw::table::ExposureCatalog::const_iterator i = subcat.begin(); i != subcat.end(); ++i) {
```

Comment:
without the &&, it tries to copy the elements into i:

```
src/CoaddPsf.cc:269:15: error: call to implicitly-deleted copy constructor of 'lsst::afw::table::ExposureRecord'
    for (auto exposureRecord : subcat) {
```

I'll change all the loops over the catalog to  c++11 style, in a separate commit.